docs/data-guidelines.md

```diff
@@ -177,6 +177,12 @@ However, this guideline does not apply to features where the browser's expected
 
 This guideline was proposed in [#6906](https://github.com/mdn/browser-compat-data/issues/6906).
 
+## Constants
```

Comment:
Oh, I missed one thing: can you update the ToC at the top of this file? Line 5 has instructions for doing this easily-ish

Reply:
Will do

Reply:
> Oh, I missed one thing: can you update the ToC at the top of this file? Line 5 has instructions for doing this easily-ish

OK, ToC updated